doc/source/metadata/Probabilities.rst

```diff
@@ -0,0 +1,346 @@
+processing to a variable
+(e.g. a maximum of the temperature over a period of time).
+This is represented as a string attribute comprising
+a list of blank-separated words of the form ``name: method``. 
+Each ``name: method`` pair indicates that for an axis identified by ``name``,
+the values representing the variable have been determined
+or derived by the specified ``method``.
+
+name
+    Can be a dimension of the variable, a scalar coordinate variable,
+    a valid `CF Standard Name`_, or the word ``area``. 
+
+method
+    Should be selected from a list:
+    ``point``,
+    ``maximum``,
+    ``maximum_absolute_value``,
+    ``median``,
+    ``mid_range``,
+    ``minimum``,
+    ``minimum_absolute_value``,
+    ``mean, mean_absolute_value``,
+    ``mean_of_upper_decile``,
+    ``mode``,
+    ``range``,
+    ``root_mean_square``,
+    ``standard_derivation``,
+    ``sum, sum_of_squares``,
+    ``variance``.
+
+If any method other than ``point`` is specified for a given axis,
+then bounds should also be provided for that axis.
+(``point`` is essentially no statistical processing applied on that axis,
+and is unusally omitted, but could be included if it improver clarity.)
+For example, a one-dimensional array of maximum air temperatures,
+could be represented as:
+
+.. code-block:: python
+
+    variables: 
+        float air_temperature(time) ;
+            air_temperature:cell_methods = "time: maximum" ;
+        int64 time ;
+            time:bounds = "time_bnds" ;
+            time:units = "seconds since 1970-01-01 00:00:00" ;
+            time:standard_name = "time" ;
+            time:calendar = "gregorian" ;
+	int64 time_bnds(bnds) ;
+
+The ``time_bnds`` values define the periods over which 
+the statistical processing has been calculated. 
+If more than one cell method is to be indicated,
+they should be arranged in the order they were applied.
+The left-most operation is assumed to have been applied first. 
+
+It is possible to indicate more precisely how the cell method was applied
+by including extra information in parentheses after the method.
+This information includes standardized and non-standardized parts.
+The only standardized option is ``interval``,
+used to provide the typical interval between the original data values
+to which the method was applied,
+in the situation where the present data values are statistically representative 
+of original data values which had a finer spacing.
+
+.. warning::
+
+    It is important to understand that the following example, 
+    does not (necessarily) represent a 1-hour maximum temperature,
+    as the period over which the maximum is derived is included in the bounds
+    (not shown), but rather that the temperature data used to calculate
+    the maximum was provided in 1-hour steps.
+
+        .. code-block:: python
+
+            air_temperature: cell_methods = "time: maximum (interval: 1 hour)" ;
+
+The non-standardised part is included in the same way,
```

Comment:
It would be, but as there is only one standardised part, `interval`, which I'm trying to stop people using here, I didn't wantto. However, I've now added some text to point the reader to the CF Convention and re-structured to read a bit better (I hope).